### DISCUSSION TITLE
It would be cool if there was a setting to turn on or off commands

### DISCUSSION THREAD

**Discussion by gugollo** [ThibautPlg/Discord-LEGO-Bot#8] [Ideas]
It would be useful to have a setting to allow the user wether he would want the whole "!set" or "!part" and it would just send the infos when the number of the set or the part is sent.

**Comment by ThibautPlg** [comment]
Hi,
Thanks for your message. 
I am not sure to understand, you would like to be able to ask the bot to show sets/parts without using the prefix trigger ? If so, how would the bot be able to differentiate a query for a set or, for example, someone mentioning a date in the chat ?
